tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -218,5 +221,39 @@ internal static string GetLogDirectory()
 
             return logDirectory;
         }
+
+        internal static void CleanLogFiles(string logsDirectory)
+        {
+            var logDaysLimit = EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.LogFileRetentionDays) ?? "32";
+
+            if (int.TryParse(logDaysLimit, out var days) && days > 0)
```

Comment:
If the env var has an invalid value, this `if` will disable the clean up. Should we fallback to the default value (32) instead? 🤔

Reply:
Since there's not an additional on/off configuration, I think anything that is not a positive number would be a good signal for disabling the clean up

Reply:
In that case should I add a validation for setting the value? I believe that If they read our docs and use an unsupported value it makes sense that we don't run at all, that way we don't delete log files that they may want to keep hence why they tried giving a value to the configuration in the first place.

Reply:
Oh, good point, Zach! As it is now, at least users can disable this with `DD_TRACE_LOGFILE_RETENTION_DAYS=<any invalid value>`. Never mind, then.